.buildkite/scripts/release.sh

```diff
@@ -44,6 +44,7 @@ release() {
   git checkout master && git pull
 
   yarn install --frozen-lockfile
+  yarn build
```

Comment:
I'm not sure if `build` is required here, or whether `lerna publish` will take care of all the type generation for us.